experiments/base_experiment.py

```diff
@@ -379,6 +380,14 @@ def _init_optimizer(self, param_groups=None):
                 betas=self.cfg.training.betas,
                 weight_decay=self.cfg.training.weight_decay,
             )
+        elif self.cfg.training.optimizer == "LookAhead":
```

Comment:
I think there is a more scalable way of doing this:

We add a boolean parameter `training.use_lookahead` to the config, and after the if-chain here we have 
```
if self.cfg.training.use_lookahead:
    self.optimizer = torch_optimizer.Lookahead(self.optimizer, k=self.cfg.training.lookahead_k, self.cfg.training.lookahead_alpha)
```
For the loading and saving, we would change `self.cfg.training.optimizer=='LookAhead'` -> `self.cfg.training.use_lookahead`.